File: C#/백준/Bronze/17210. 문문문/문문문.cs
using System;
using System.Numerics;

namespace q{
    class w{
        static void Main(){
            BigInteger q = BigInteger.Parse(Console.ReadLine());
            if(q>=6){
                Console.WriteLine("Love is open door");
            }
            else{
                int w = int.Parse(Console.ReadLine());
                if(w==0){
                    for (int i=0; i<q-1; i++){
                        Console.WriteLine(1-i%2);
                    }
                }
                else{
                    for (int i =0; i<q-1; i++){
                        Console.WriteLine(i%2);
                    }
                }
            }
        }
    }
}

File: Java/백준/Bronze/17210. 문문문/문문문.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String q = in.next();
        if(q.length()>1){
            System.out.println("Love is open door");
        }
        else{
            int w = Integer.parseInt(q);
            if(w>=6){
                System.out.println("Love is open door");
            }
            else{
                int e = in.nextInt();
                if(e==0){
                    for (int i=0; i<w-1;i++){
                        System.out.println(1-i%2);
                    }
                }
                else{
                    for (int i =0; i<w-1; i++){
                        System.out.println(i%2);
                    }
                }
            }
        }
    }
}